posts/2020-10-05-Rust-1.47.md

```diff
@@ -0,0 +1,180 @@
+stack backtrace:
+   0: std::panicking::begin_panic
+             at /rustc/d6646f64790018719caebeafd352a92adfa1d75a/library/std/src/panicking.rs:497
+   1: playground::main
+             at ./src/main.rs:2
+   2: core::ops::function::FnOnce::call_once
+             at /rustc/d6646f64790018719caebeafd352a92adfa1d75a/library/core/src/ops/function.rs:227
+```
+
+This makes it much easier to see where the panic actually originated, and
+you can still set `RUST_BACKTRACE=full` if you want to see everything.
+
+#### LLVM 11
+
+We have [upgraded to LLVM 11](https://github.com/rust-lang/rust/pull/73526/).
+The compiler still supports being compiled with LLVM versions as old as 8,
+but by default, 11 is what you'll be getting.
+
+#### Control Flow Guard on Windows
+
+`rustc` [now supports](https://github.com/rust-lang/rust/pull/73893/) `-C
+control-flow-guard`, an option that will turn on [Control Flow
+Guard](https://docs.microsoft.com/en-us/windows/win32/secbp/control-flow-guard)
+on Windows. Other platforms ignore this flag.
+
+### Library changes
+
+Nine new APIs were stabilized this release:
+
+- [`Ident::new_raw`]
+- [`Range::is_empty`]
+- [`RangeInclusive::is_empty`]
+- [`Result::as_deref`]
+- [`Result::as_deref_mut`]
+- [`Vec::leak`]
+- [`pointer::offset_from`]
+- [`f32::TAU`]
+- [`f64::TAU`]
+
+Additonally, the following previously stable APIs have now been made `const`:
+
+- [The `new` method for all `NonZero` integers.][73858]
+- [The `checked_add`,`checked_sub`,`checked_mul`,`checked_neg`, `checked_shl`,
+  `checked_shr`, `saturating_add`, `saturating_sub`, and `saturating_mul`
+  methods for all integers.][73858]
+- [The `checked_abs`, `saturating_abs`, `saturating_neg`, and `signum`  for all
+  signed integers.][73858]
+- [The `is_ascii_alphabetic`, `is_ascii_uppercase`, `is_ascii_lowercase`,
+  `is_ascii_alphanumeric`, `is_ascii_digit`, `is_ascii_hexdigit`,
+  `is_ascii_punctuation`, `is_ascii_graphic`, `is_ascii_whitespace`, and
+  `is_ascii_control` methods for `char` and `u8`.][73858]
+
+[`Ident::new_raw`]:  https://doc.rust-lang.org/nightly/proc_macro/struct.Ident.html#method.new_raw
+[`Range::is_empty`]: https://doc.rust-lang.org/nightly/std/ops/struct.Range.html#method.is_empty
+[`RangeInclusive::is_empty`]: https://doc.rust-lang.org/nightly/std/ops/struct.RangeInclusive.html#method.is_empty
+[`Result::as_deref_mut`]: https://doc.rust-lang.org/nightly/std/result/enum.Result.html#method.as_deref_mut
+[`Result::as_deref`]: https://doc.rust-lang.org/nightly/std/result/enum.Result.html#method.as_deref
+[`TypeId::of`]: https://doc.rust-lang.org/nightly/std/any/struct.TypeId.html#method.of
+[`Vec::leak`]: https://doc.rust-lang.org/nightly/std/vec/struct.Vec.html#method.leak
+[`f32::TAU`]: https://doc.rust-lang.org/nightly/std/f32/consts/constant.TAU.html
+[`f64::TAU`]: https://doc.rust-lang.org/nightly/std/f64/consts/constant.TAU.html
+[73858]: https://github.com/rust-lang/rust/pull/73858/
+
```

Comment:
Should this mention adding the Ayu theme to rustdoc? That seems like a pretty large change to me.

Reply:
I don't think it should be the highlight of the release, but we should mention it.